packages/jaeger-ui/src/components/Monitor/ServicesView/serviceGraph.tsx

```diff
@@ -40,16 +40,11 @@ type TProps = {
   yDomain?: number[];
   color?: string;
   marginClassName?: string;
-  yAxisTickFormat?: (v: number) => number;
+  yAxisTickFormat?: (v: number) => string;
```

Comment:
Can't we return a number here?

Reply:
Nope, as I previously described that timeFormatter function only accept string.